bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -353,6 +353,7 @@
 
 		<channels>
 			<channel id="color" typeId="system.color"/>
+			<channel id="color_temperature" typeId="system.color-temperature"/>
```

Comment:
Please update the default translation file to add your new channel.

As you are adding a new channel, I assume you should use the new thing upgrade mechanism to trigger the automatic upgrade of things (things created with UI).

Reply:
Sorry, I'm new here. Do you mand adding the channel like it is done for e.g. the rollershutter here:
https://github.com/openhab/openhab-addons/blob/6e0fe0e0ad3b501977d8c7ea4e7b21fce14749b1/bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/i18n/avmfritz.properties#L183

I'm wondering a bit, because the `color` channel is missing in that file.

Reply:
> As you are adding a new channel, I assume you should use the new thing upgrade mechanism to trigger the automatic upgrade of things (things created with UI).

fixed with the hint from @jlaur